src/includes/init.php

```diff
@@ -36,6 +38,13 @@ function redirect($url, $flash = null)
     exit();
 }
 
+function error($title, $body)
+{
+    $_SESSION['errtitle'] = $title ?? "fatal error";
+    $_SESSION['errcode'] = $body ?? "fatal error";
+    redirect("/includes/errorPage/errorPage.php");
+}
+
 function getFlash()
```

Comment:
My `getFlash` function has a few issues. It can't be called more than once for one. Also I think I want to be able to tell it what I want to fetch. Like `getFlash('errtitle');`.

Reply:
I'm going to leave it as is for now then. can always change it over later